src/nvim/api/vim.c

```diff
@@ -1562,3 +1563,15 @@ Object nvim_get_proc(Integer pid, Error *err)
 #endif
   return rvobj;
 }
+
+Array nvim__inspect(Integer row, Integer col)
```

Comment:
though this is "private", it could still benefit from a more intuitive name, maybe `nvim__screen_inspect` or `nvim__ui_inspect`

Reply:
Currently it is mostly for my own debugging, it will be made redundant or replaced with something better.